app/gui2/package.json

```diff
@@ -124,15 +118,14 @@
     "@vue/eslint-config-typescript": "^13.0.0",
     "@vue/test-utils": "^2.4.6",
     "@vue/tsconfig": "^0.5.1",
-    "change-case": "^4.1.2",
     "cross-env": "^7.0.3",
     "css.escape": "^1.5.1",
     "d3": "^7.4.0",
     "enso-common": "workspace:*",
     "eslint": "^8.49.0",
     "eslint-plugin-vue": "^9.22.0",
     "floating-vue": "^2.0.0-beta.24",
-    "hash-wasm": "^4.10.0",
+    "hash-wasm": "^4.11.0",
```

Comment:
Yes, but it is a bit more complicated to do than it may seem. We have to check what's the performance difference between the two. I'd stick with `@noble/hashes` if it isn't significantly slower than wasm. Otherwise we want to keep the wasm version, but that complicates things a little bit due to our polyglot (engine) ydoc-server environment not supporting wasm modules.